### ISSUE TITLE
Add 'old' Tag for Last Version

### ISSUE THREAD

**Issue by surmann** [rocker-org/rocker#341]
It would be nice to have a tag for the last R version.

I test my package against 'devel', 'latest', and '3.4'. Progression to R 3.6 we have to update all our yml files from tag '3.4' to '3.5'. Here it would help to have the new tag 'old', for instance.

**Comment by nuest** [comment]
In line with CRAN, this tag could be `oldrel`, and one can probably add this to the [`post_push` tagging mechanism](https://github.com/rocker-org/rocker-versioned/blob/master/r-ver/3.5.3/hooks/post_push).

**Comment by cboettig** [comment]
Thanks, keep meaning to get back to this but haven't done it.  agree we just need a post_push hook to do this.  @nuest want to send a PR for this?

**Comment by surmann** [comment]
Is it easy to write a peace of code which rolls the `oldrel` version forward related to the current version of R?

**Comment by cboettig** [comment]
@surmann Yes, I think we just need to add the relevant logic to the post_push hook to create and push the extra tag.  See https://github.com/rocker-org/rocker-versioned/blob/master/r-ver/hooks/post_push and the Docker docs on hooks.  -- PRs welcome.  Ideally this could be done in a way that doesn't require the script to be updated manually on each new R release...
